local_libs/lib-dex-commons/src/main/kotlin/gov/cdc/dex/mmg/MmgUtil.kt

```diff
@@ -63,7 +63,7 @@ class MmgUtil(val redisProxy: RedisProxy)  {
     @Throws(InvalidConditionException::class)
     // Populates DexMessageInfo, including list of MMGs and provision route.
     fun getMMGMessageInfo(msh21_2: String, msh21_3: String?, eventCode: String, jurisdictionCode: String?): DexMessageInfo {
-        val messageInfo = DexMessageInfo(eventCode, null, null, jurisdictionCode)
+        val messageInfo = DexMessageInfo(eventCode, null, null, jurisdictionCode, "ECR")
```

Comment:
Can we use the ENUM value here instead of hard coded string?

Reply:
I think that's a great idea... I can make that change quickly... 
Also, i'm commenting out the type on Condition2MMG for now - I'm rethinking this and not sure it will work there.

Reply:
Sounds good Marcelo.